src/main/java/org/folio/circulation/domain/AddressType.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.circulation.domain;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+import io.vertx.core.json.JsonObject;
+
+
+public class AddressType {
+  private final String addressType;
+  /**
+   * Address Type description
+   */
+  private final String desc;
```

Comment:
@andrii-paias, is it possible to rename the property instead of adding a comment :)?

Reply:
Thanks, updated PR